server/bin/pbench-dispatch.sh

```diff
@@ -8,9 +8,9 @@
 # for processing (`pbench-agent` v0.51 and later).  It verifies the tar balls
 # and their MD5 check-sums and then moves the tar balls to the archive tree,
 # setting up the appropriate state links (e.g. a production environment may
-# want the tar balls unpacked (TO-UNPACK), backed up (TO-BACKUP), and indexed
-# (TO-INDEX), while a satellite environment may only need to prepare the
-# tar ball to be synced to the main server (TO-SYNC)).
+# want the tar balls unpacked (TO-UNPACK), and indexed (TO-INDEX), while a
+# satellite environment may only need to prepare the tar ball to be synced to
+# the main server (TO-SYNC)).
```

Comment:
No comment about backing them up after verifying them and before moving them to the archive tree?

(And, you don't need that comma before "and indexed".)

Reply:
Fixed.